docker/dockerfiles/bedrock_linkchecker

```diff
@@ -0,0 +1,21 @@
+FROM debian:jessie
+
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential python python-dev python-pip
+
+RUN useradd -ms /bin/bash linkchecker
+COPY ./requirements /home/linkchecker/app/requirements
+COPY ./bin /home/linkchecker/app/bin
+
+RUN chown -R linkchecker:linkchecker /home/linkchecker
+
+WORKDIR /home/linkchecker/app
+
+RUN mkdir /results && chown -R linkchecker:linkchecker /results
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
+RUN pip install -r requirements/pip.txt
+RUN ./bin/peep.py install --no-cache-dir -r requirements/linkchecker.txt
+
+USER linkchecker
```

Comment:
The LinkChecker tool detects if it's running as root and drops to a 'nobody' user, which means it doesn't have the necessary privileges to create the results files. Using a 'linkchecker' user with appropriate permissions gets around this.